sdks/crdbase/crd.go

```diff
@@ -25,7 +25,7 @@ var (
 )
 
 // NewCustomResourceDefinition returns a new CRD object.
-func (crdb *CRDBase) NewCustomResourceDefinition(
+func (crdb *CrdBase) NewCustomResourceDefinition(
```

Comment:
`CRDBase` means `custom resource data`base ((not Definition)) so all capital letters will be better?

Reply:
done.